Makefile

```diff
@@ -590,7 +590,9 @@ ifeq "$(INSTALL_OCAMLNAT)" "true"
 	  $(INSTALL_PROG) ocamlnat$(EXE) "$(INSTALL_BINDIR)"
 endif
 	cd "$(INSTALL_COMPLIBDIR)" && \
-	   $(RANLIB) ocamlcommon.$(A) ocamlbytecomp.$(A) ocamloptcomp.$(A)
+	   printf "ocamlcommon.$(A)\nocamlbytecomp.$(A)\nocamloptcomp.$(A)" |\
+	   xargs -I{} $(RANLIB) {}
```

Comment:
Let's just keep it simple and run `$(RANLIB)` on each of the archives, no need for `printf` and `xargs`.